catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -316,11 +316,13 @@ public void shutdown() {
     }
 
     public void checkForRollover() {
+        LOGGER.trace("checking for rollover");
```

Comment:
How often does this occur? 30 times/sec? If there is no path around this call (i.e. we receive a packet and then always call this method) then I'm not sure we need this one. The places where these kinds of logging statements are helpful is when there is a conditional statement and it tells us which branch we went down. The statement in the doRollover method is a good example, it tells us when we actually roll the file (we can assume (as long as we're receiving packets) that all the rest of the times this is called the check for roll over didn't pass.